src/rcprg_smach/human_fell.py

```diff
@@ -1,13 +1,13 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # encoding: utf8
 
 import math
 import random
-import rospy
+import rclpy
 import smach
 import smach_ros
 import dynamic_reconfigure.client
```

Comment:
Do sprawdzenia czy wykorzystywane, chyba nie, więc jeśli potwierdzisz, że nie wykorzystywane to usunąć